components/afObjectField/afObjectField.js

```diff
@@ -3,5 +3,10 @@
 Template.afObjectField.helpers({
   getTemplateName: function () {
     return AutoForm.getTemplateName('afObjectField', this.template, this.name);
+  },
+  innerContext: function() {
+    var c = AutoForm.Utility.getComponentContext(this, 'afObjectField');
+    _.extend(this, c.atts);
+    return this;
```

Comment:
Thanks for pointing this out.

I've pushed a version using the first version, at I cannot assume, how this is used in the templates used by other users.